Plugins/Gradle/build.gradle

```diff
@@ -97,6 +98,15 @@ task sourcesJar(type: Jar) {
 apply from: "../../publish.build.gradle"
 apply from: '../../ktlint.gradle'
 
+gradlePlugin {
+    plugins {
+        simplePlugin {
+            id = "dev.testify"
+            implementationClass = "dev.testify.TestifyPlugin"
```

Comment:
I tested this out over the weekend and it seems to be generating 3 separate entries and was unpublishable to Maven Central.